frontend/src/components/Nav/ClusterMenu/ClusterMenu.tsx

```diff
@@ -27,18 +20,17 @@ interface ClusterMenuProps {
   name: Cluster['name'];
   status: Cluster['status'];
   features: Cluster['features'];
-  singleMode?: boolean;
+  openTab?: string | false;
```

Comment:
```suggestion
  openTab?: string;
```

do we need the `false` can't we treat `undefined` as false

Reply:
and maybe make the tabs as a `enums`  in terms of object with `as const` so that we don't pass any arbitrary string

Reply:
@Mgrdich these are cluster names, not the menu items, so an enum is not applicable
`handleTabChange(cluster.name)}`